codespell_lib/data/dictionary.txt

```diff
@@ -24283,6 +24334,7 @@ plaforms->platforms
 plaftorm->platform
 plaftorms->platforms
 plagarism->plagiarism
+plaint->plain, paint,
```

Comment:
Checks failed `plaint` is a valid word. I suggest you remove this.
* https://www.google.com/search?q=define+plaint
* https://www.lexico.com/definition/plaint
* https://www.ldoceonline.com/dictionary/plaint